README.md

```diff
@@ -11,26 +11,47 @@
   - [classification](https://github.com/stac-extensions/classification/)
   - [eo](https://github.com/stac-extensions/eo)
   - [item-assets](https://github.com/stac-extensions/item-assets)
+  - [mgrs](https://github.com/stac-extensions/mgrs)
   - [proj](https://github.com/stac-extensions/projection)
   - [raster](https://github.com/stac-extensions/raster)
   - [scientific](https://github.com/stac-extensions/scientific)
+  - [view](https://github.com/stac-extensions/view)
 - Extra fields:
-  - `hls:tile-id`: Tile identifier using the Military Grid Reference System (MGRS)
+  - `hls:product`: "HLSL30" (Landsat) or "HLSS30" (Sentinel) product
```

Comment:
What are your thoughts on implementing a custom STAC extension, ala https://github.com/stactools-packages/noaa-mrms-qpe/tree/main/extension? Too much work for too little payoff?

Reply:
See #2